site/src/views/Status/transform.js

```diff
@@ -59,5 +115,7 @@ const aggregateAlertLevel = (items = []) =>
 
 export {
   createProjectionStatusModel,
+  createImportStatusModel,
+  createCachestatusModel,
```

Comment:
```suggestion
  createCacheStatusModel,
```

`createCacheStatusModel`, capital `S`